pkg/api/kubegraph/nodes/nodes.go

```diff
@@ -130,8 +130,10 @@ func EnsurePodTemplateSpecNode(g osgraph.MutableUniqueGraph, ptSpec *kapi.PodTem
 		},
 	).(*PodTemplateSpecNode)
 
-	podSpecNode := EnsurePodSpecNode(g, &ptSpec.Spec, ptSpecName)
-	g.AddEdge(ptSpecNode, podSpecNode, osgraph.ContainsEdgeKind)
+	if ptSpec != nil {
```

Comment:
Based on the changes here, I'd bet on it being this one: https://github.com/openshift/origin/pull/6246/files#r47775508